pyproject.toml

```diff
@@ -40,8 +41,7 @@ comparefiles = "linkml_runtime.utils.comparefiles:cli"
 linkml-normalize = "linkml_runtime.processing.referencevalidator:cli"
 
 [tool.poetry.dependencies]
-python = "^3.7.6"
-
+python = "^3.9.0"
```

Comment:
Is there a reason this needs to be 3.9 instead of 3.8? Just curious since 3.8 won't reach end-of-life until [later this year](https://devguide.python.org/versions/), and I think we should try and keep this package as "installable" as possible.

Reply:
Upgrading Prefixmaps to a version that [requires Python to be 3.9 or higher](https://github.com/linkml/prefixmaps/blob/main/pyproject.toml#L27), implicitly implies that 3.8 won't be supported anymore.

That version of Prefixmaps is needed to have support for Python 3.12. Therefore as I see it, it's XOR decision between supporting 3.8 and supporting 3.12. I haven't found any other way to have support for Python 3.12.

Reply:
@pkalita-lbl can we mark this conversation as resolved? IMO the posterior changes on the branch to take advantage of your fixes in `prefixmaps` should be sufficient.